javascript-modal/styles.css

```diff
@@ -0,0 +1,56 @@
+h2 {
+  margin-bottom: 4rem;
+}
+
+.overlay {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.742);
+}
+
+.open-modal {
+  padding: 0.5rem 2rem;
+  font-size: 22px;
+  color: white;
+  background-color: rgb(80, 80, 254);
+  border-radius: 5px;
+}
+
+.open-modal:hover {
+  cursor: pointer;
+}
+
+.modal {
+  position: absolute;
+  top: 30%;
```

Comment:
This seems like a really manual way to center out your content, and it will only look correct on certain screen sizes. Is there a way you can center this so that it will be centered on all screen sizes?

Reply:
Would it be flex ? I couldn't figure out how to use flex to center in viewport so I used this ..